templates/master.yaml

```diff
@@ -512,31 +411,26 @@ coreos:
     content: |
       [Unit]
       Description=etcd2
-      Requires=k8s-setup-network-env.service
-      After=k8s-setup-network-env.service
+      # TODO can we move this to a Wants in the decrypt-tls-certs.service ?
+      Requires=k8s-setup-network-env.service decrypt-tls-certs.service
```

Comment:
I thought the `decrypt-tls-certs.service` is AWS specific and thus has to be injected via the extensions API. A dependency the other way around then might work. So `decrypt-tls-certs.service` has a `Wants` and an `After` on the `k8s-setup-network-env.service` if this is necessary.